### PR TITLE
Update agent readmes to be universal

### PR DESCRIPTION
**Pull request by 5A11**
## Proposed Changes

Update agents' readmes to be universal across Github and Agentverse

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Bug fix
- [ ] New agent
- [ ] New solution
- [ ] New application

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/uAgents/blob/main/CONTRIBUTING.md) guide 
- [x] Checks and tests pass locally

### CODE REVIEW

File: 1-uagents/finance/company-ticker-resolver-agent/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
...
+[![live](https://img.shields.io/badge/Live-8A2BE2?logo=elixir
+)](https://agentverse.ai/)
```
Comment (5A11) [inline 1-uagents/finance/company-ticker-resolver-agent/README.md:6]: Is this not deployed? Where is the URL for this?

File: 1-uagents/finance/finance-q&a-agent/README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
+[![live](https://img.shields.io/badge/Live-8A2BE2?logo=elixir
+)](https://agentverse.ai)
```
Comment (5A11) [inline 1-uagents/finance/finance-q&a-agent/README.md:7]: Same as above.

File: 1-uagents/geo/open-charge-map-agent/README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
+[![live](https://img.shields.io/badge/Live-8A2BE2?logo=elixir
+)](https://agentverse.ai/)
```
Comment (5A11) [inline 1-uagents/geo/open-charge-map-agent/README.md:6]: Same as above.

File: 1-uagents/travel/flights-retriever-agent/README.md [hunk 4]
```diff
@@ -2,6 +2,8 @@
...
+[![live](https://img.shields.io/badge/Live-8A2BE2?logo=elixir
+)](https://agentverse.ai/)
```
Comment (5A11) [inline 1-uagents/travel/flights-retriever-agent/README.md:6]: Same as above.

File: 1-uagents/utility/website-validation-agent/README.md [hunk 5]
```diff
@@ -3,6 +3,8 @@
...
+[![live](https://img.shields.io/badge/Live-8A2BE2?logo=elixir
+)](https://agentverse.ai)
```
Comment (5A11) [inline 1-uagents/utility/website-validation-agent/README.md:7]: Same as above.

**Review by jrriehl — APPROVED** [review]

**Comment by Archento** [comment]
The agents weren't deployed at that time so we need to update the links once they all are. I guess this will be ironed out once the deployment pipeline is solid and contents in the source repo are correct.

**Pull request merged**